README.md

```diff
@@ -1 +1,18 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+Built With
+.Html
+.CSS
+.Javascript
+
+Getting Started
+ .pull the project then run it with broweser
+Prerequisites
+ .Use browser
+ .Install
+Authors
+ .GitHub: @nati2323
+Give a ⭐️ if you like this project!
+License
+ .This project is MIT licensed.
+
+
```

Comment:
- [ ] [REQUIRED] Great job in adding the README file, however, as stated in the project requirements, please use the [Microverse README Template](https://github.com/microverseinc/readme-template/blob/master/README.md) but remember to customize it to your own project. If you don't know how you can add the titles and all that, when you visit the page given, click on raw and copy the template but remember to customize it to your own project.

- You can add the sections such as in the **Authors** you can add your twitter and linkedin link, also sections that apply such as **Contributing**, and **Show your support** 😊😊

Reply:
I did the correction. Thanks!